robottelo/cli/base.py

```diff
@@ -374,6 +374,13 @@ def _construct_command(cls, options=None):
                 if val is True:
                     tail += u' --{0}'.format(key)
                 elif val is not False:
+                    if isinstance(val, list):
+                        val = ','.join(
+                            str(el.get('id') or el.get('name'))
```

Comment:
@Ichimonji10 again, that line is for dictionaries. If you wanna pass labels, UUIDs, then you're doing it like so:
```python
'hostgroups': [hg['label'] for hg in new_host_groups],
```